src/hyperlight_host/src/mem/shared_mem.rs

```diff
@@ -82,6 +92,8 @@ macro_rules! generate_writer {
 pub struct HostMapping {
     ptr: *mut u8,
     size: usize,
+    #[cfg(target_os = "windows")]
+    handle: HandleWrapper,
```

Comment:
question: is it possible to put a raw HANDLE here and remove our wrapper?

Reply:
Yes, fixed